genkit-tools/package.json

```diff
@@ -33,6 +33,9 @@
     "url": "https://github.com/firebase/genkit.git",
     "directory": "genkit-tools"
   },
+  "engines": {
+    "node": ">=20.0.0"
```

Comment:
Is it only a requirement in dev or runtime as well?

Reply:
I believe runtime as well, Jonathan and Ian were getting those errors when trying to use the CLI runner js with Node v18.

Reply:
Maybe we just remove this though and people can try with lower versions if they want (with the caveat that it might not work). I can re-phrase the contributing.md change for this.